### PR TITLE
[TACHYON-550]: Read Tachyon version from TachyonConf

### PR DESCRIPTION
**Pull request by miasantreble**

### CODE REVIEW

File: common/src/main/java/tachyon/Version.java [hunk 1]
```diff
@@ -31,25 +30,8 @@
   private static final Logger LOG = LoggerFactory.getLogger(Constants.LOGGER_TYPE);
```
Comment (calvinjia) [inline common/src/main/java/tachyon/Version.java]: I think you can remove the logger now?

**Comment by calvinjia** [comment]
Thanks for the patch, one minor comment. Also, could you put the ticket in brackets (`[TACHYON-550]`)?

File: common/src/main/java/tachyon/Version.java (hunk 1 above)
Comment (miasantreble) [inline common/src/main/java/tachyon/Version.java]: done

File: common/src/main/java/tachyon/Version.java [hunk 2]
```diff
@@ -28,28 +24,9 @@
...
-          in.close();
-        } catch (Exception e) {
-          LOG.error(e.getMessage(), e);
-        }
-      }
-    }
-
-    VERSION = p.getProperty("tachyon.version", "UNDEFINED");
+    TachyonConf tachyonConf = new TachyonConf();
+    VERSION = tachyonConf.get(Constants.TACHYON_VERSION, null);
```
Comment (gpang) [inline common/src/main/java/tachyon/Version.java]: Should the default match the previous default of "UNDEFINED"?

Comment (miasantreble) [inline common/src/main/java/tachyon/Version.java]: Thanks for the reminder! Fixed now

**Comment by apc999** [comment]
+1 LGTM. Thanks a lot for the path! It really simplifies the code greatly.

**Comment by haoyuan** [comment]
Thanks!

**Pull request merged**